lib/channel.ts

```diff
@@ -84,15 +84,23 @@ function createSender(sourceId: string, targetWindow: Window) {
   return function send(method: string, params: any) {
     const messageId = messageCount++
 
-    targetWindow.postMessage(
-      {
-        source: sourceId,
-        id: messageId,
-        method,
-        params,
-      },
-      '*'
-    )
+    try {
+      targetWindow.postMessage(
+        {
+          source: sourceId,
+          id: messageId,
+          method,
+          params,
+        },
+        '*'
+      )
+    } catch (e) {
+      if ((e as Error).name === 'DataCloneError') {
```

Comment:
nit as I just saw this PR:

Instead of assuming the `Error` type, you can simply check for `e instanceof DOMException`. Then you can also check for `DataCloneError`.

Would make my inner TypeScript monk happy :D

Reply:
Done ;)